presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayUnionFunction.java

```diff
@@ -91,4 +103,28 @@ private static void appendBigintArray(Block array, AtomicBoolean containsNull, L
             }
         }
     }
+
+    private static void appendTypedArray(Block array, TypedSet typedSet)
```

Comment:
please move this back where it was, i.e. right below generic `union`

Reply:
I organized the functions to be public methods first then private methods later. Do you think it's better to make each private helper method after its caller? Do we have a general guideline how to organize them?